lib/PasswordValidator.php

```diff
@@ -73,8 +75,19 @@ public function validate(string $password): void {
 				continue;
 			}
 
-			$instance->validate($password);
+			try {
+				$instance->validate($password);
+			} catch (HintException $e) {
+				$errors[] = $e->getMessage();
+				$hints[] = $e->getHint();
+			}
 		}
-	}
 
+		if (!empty($errors)) {
+			throw new HintException(
+				implode(' ', $errors),
+				implode(' ', $hints),
```

Comment:
If we go with `\n` here, it would look better in the CLI, but the frontend doesn't understand that, so it would need fixing in the server as well to transform `\n` into `<br>` and allow html in first place

Reply:
Fine for now I'd say. Maybe we need a different kind of exception for this.